prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -477,4 +477,45 @@ periodics:
       resources:
         requests:
           memory: 200Mi
-          cpu: 80m
\ No newline at end of file
+          cpu: 80m
+- name: kyma-load-tests-weekly
+  cron: "0 2 * * 1" # "At 2:00 on every Monday."
+  labels:
+    preset-build-master: "true"
+    <<: *gke_job_labels_template
+  decorate: true
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+    - <<: *kyma_ref
+      base_ref: master
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190129-c951cf2
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        env:
+          - name: INPUT_CLUSTER_NAME
+            value: "load-test"
+          - name: SLACK_CLIENT_TOKEN
```

Comment:
probably you should use  preset-stability-checker-slack-notifications

Reply:
FYI: `preset-stability-checker-slack-notifications` should not be used anymore as it's legacy approach for sending msg to Slack (jenkins webook was a hack, now we should use the Slack Bot), so the `preset-sap-slack-bot-token` is a proper approach